### DISCUSSION TITLE
文字识别模型返回单字字符时，如果对图片进行不同处理，有时返回的字数总数会有差异，如何确保总字数一致，方便对比单字结果？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#13675] [Ideas & Features]
文字识别模型返回单字字符时，如果对图片进行不同处理，有时返回的字数总数会有差异，如何确保总字数一致，方便对比单字结果？

**Comment by UserWangZz** [comment]
模型预测结果没办法控制啊

**Comment by nissansz** [comment]
主要是会多字符或漏字符，有没有一些办法改善？








--------------------------------------------------------------------------------





------------------ 原始邮件 ------------------




 
 发件人: WangZ  ***@***.***>
 

 
 发送时间: 2024-08-19 12:26:01
 

 
 收件人:PaddlePaddle/PaddleOCR ***@***.***> 

 
 抄送:nissanjp ***@***.***>,Author ***@***.***> 

 
 主题: Re: [PaddlePaddle/PaddleOCR] 文字识别模型返回单字字符时，如果对图片进行不同处理，有时返回的字数总数会有差异，如何确保总字数一致，方便对比单字结果？ (Discussion #13675)
 





 模型预测结果没办法控制啊
